src/main/java/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,67 @@
+package bookstore.controller;
+
+import bookstore.dto.BookDto;
+import bookstore.dto.BookSearchParametersDto;
+import bookstore.dto.CreateBookRequestDto;
+import bookstore.service.BookService;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Positive;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@Validated
+@RequestMapping("/books")
+public class BookController {
+    private final BookService bookService;
+
+    @ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
this is a wrong status for this endpoint

see the explanation of HTTP 202:
`The HTTP status code 202 is used to indicate that a request has been accepted for processing, but the processing has not yet been completed.`

consider using HTTP 200 here